hedera-node/src/main/java/com/hedera/services/fees/calculation/crypto/queries/GetAccountRecordsResourceUsage.java

```diff
@@ -42,34 +40,35 @@
 
 	@Inject
 	public GetAccountRecordsResourceUsage(
-			AnswerFunctions answerFunctions,
-			CryptoFeeBuilder usageEstimator
+			final AnswerFunctions answerFunctions,
+			final CryptoFeeBuilder usageEstimator
 	) {
 		this.answerFunctions = answerFunctions;
 		this.usageEstimator = usageEstimator;
 	}
 
 	@Override
-	public boolean applicableTo(Query query) {
+	public boolean applicableTo(final Query query) {
 		return query.hasCryptoGetAccountRecords();
 	}
 
 	@Override
-	public FeeData usageGiven(Query query, StateView view) {
+	public FeeData usageGiven(final Query query, final StateView view) {
 		return usageGivenType(query, view, query.getCryptoGetAccountRecords().getHeader().getResponseType());
 	}
 
 	@Override
-	public FeeData usageGivenType(Query query, StateView view, ResponseType type) {
-		var target = fromAccountId(query.getCryptoGetAccountRecords().getAccountID());
+	public FeeData usageGivenType(final Query query, final StateView view, final ResponseType type) {
+		final var op = query.getCryptoGetAccountRecords();
+		final var target = fromAccountId(op.getAccountID());
 		if (!view.accounts().containsKey(target)) {
 			/* Given the test in {@code GetAccountRecordsAnswer.checkValidity}, this can only be
 			 * missing under the extraordinary circumstance that the desired account expired
 			 * during the query answer flow (which will now fail downstream with an appropriate
 			 * status code); so just return the default {@code FeeData} here. */
 			return FeeData.getDefaultInstance();
 		}
-		List<TransactionRecord> records = answerFunctions.accountRecords(view, query);
+		final var records = answerFunctions.accountRecords(view, op);
```

Comment:
Wanted to ask a question if the use of `var` should be favored over the use of static type declarations?

Reply:
This is opinion-based and in general I would say yes, especially since we are using Java, a statically typed programming language.
Pros:
- Shorten the code
- Most of the time we don't lose context (in this particular example, we know that we will get some records, based on the variable name and the function name)
- No need to change the type in the future when the function returns a different type (with some notes in the Cons)

Cons:
- This will fail when we have a `contract` to specifically expect a particular returned type, so we can use it downstream. But in that case, we usually add an adapter to change to the expected type, and end up being able to use `var` again for both assignments.
```
final var records = answerFunctions.accountRecords(view, op);
final var adaptedRecords = SomeAdapterFunction(records);
// Then start using adaptedRecords instead of the original records
```

Reply:
Thanks @qnswirlds , that makes sense!

Reply:
@abhishek-hedera, I forgot to mention an important case where `var` is not preferable, the diamond operator.
`List<String> stringList = new ArrayList<>();`
If you use var, `stringList` will get an inferred type of `ArrayList<Object>`, which is usually not what you wanted.